help/en/html/doc/Technical/XmlView.shtml

```diff
@@ -16,66 +16,58 @@
     <div id="mainContent">
       <h1>JMRI: Viewing XML Files</h1>
 
-      <p>JMRI stores various information in XML files. Although they're a text format that is
-      basically human-readable, it's a complicated, structured format.</p>
-
-      <p>To make this information easier to read, print and understand, most JMRI XML files can be
-      displayed in a web browser. A technique called "XSLT stylesheets" is used to format them
-      within the web browser. For an example of how this looks, please see the <a href=
-      "https://jmri.org/xml/samples/NicksClinic.xml">sample panel XML file</a> on the main JMRI web
-      site. (May take a few seconds to load)</p>
-
-      <p>To view an XML file that JMRI has created, just open the file with your favorite web
-      browser. In many cases, you should be able to just double-click on it, or drag it to the web
-      browser's icon. On some platforms, you'll need to open it from the "File" menu.</p>
-
-      <p>Unfortunately, this simple approach doesn't always work, because not all browsers properly
-      implement the algorithms for finding the formatting files that are needed for this display.
-      If the first browser you try doesn't display the XML file (usually just showing a blank page,
-      though sometimes they'll display an error message), please try other ones you've got
-      installed. Internet Explorer is particularly problematic. Mozilla, Safari and Opera seem to
-      get this right, while Firefox seems to work properly on some platforms and not on others.
-      (There might also be options that affect whether it works. Here's some <a href=
-      "https://jmri.org/xml/XSLT/README">technical background</a>)</p>
-
-      <p>If none of the browsers available to you can display the file, you can get around this
-      problem using the <a href="../../web/index.shtml">JMRI mini web server</a> in JMRI version
-      2.9.4 and later.</p>
-
-      <p>To do this:</p>
-
+      <p>JMRI stores information about its operations and information that you create in XML files. For example, when you use the JMRI "Store Files" command in PanelPro, your tables and panels (depending on which command you select) are stored in an XML file. Other XML files hold preferences, control information, operational information, decoder contents, your roster, and more. </p>
```

Comment:
Not important, but it helps if lines are no longer than 80 - 120 characters. One of the reasons is that it's easier to review PRs if the lines are not too long.

Reply:
Thanks.  I usually go back and fix lines to be more or less consistent.  Must have missed this one.